packages/cds-odata-annotation-converter/test/printer/csdl-to-cds.test.ts

```diff
@@ -0,0 +1,2399 @@
+                                value: 'Common.IsActionCritical',
+                                valueRange: {
+                                    start: { line: 49, character: 12 },
+                                    end: { line: 49, character: 35 }
+                                }
+                            },
+                            Qualifier: {
+                                type: 'attribute',
+                                name: 'Qualifier',
+                                nameRange: null,
+                                value: 'actionAfter',
+                                valueRange: {
+                                    start: { line: 49, character: 37 },
+                                    end: { line: 49, character: 48 }
+                                }
+                            }
+                        },
+                        range: {
+                            start: { line: 49, character: 12 },
+                            end: { line: 49, character: 54 }
+                        },
+                        contentRange: {
+                            start: { line: 49, character: 50 },
+                            end: { line: 49, character: 54 }
+                        },
+                        content: [
+                            {
+                                type: 'element',
+                                name: 'Bool',
+
+                                range: {
+                                    start: { line: 49, character: 50 },
+                                    end: { line: 49, character: 54 }
+                                },
+                                contentRange: {
+                                    start: { line: 49, character: 50 },
+                                    end: { line: 49, character: 54 }
+                                },
+                                content: [
+                                    {
+                                        type: 'text',
+                                        text: 'true',
+                                        range: {
+                                            start: { line: 49, character: 50 },
+                                            end: { line: 49, character: 54 }
+                                        }
+                                    }
+                                ],
+                                attributes: {}
+                            }
+                        ]
+                    }
+                ]
+            };
+
+            // act
+            const result = printTarget(target);
+
+            // assert
+            expect(result).toMatchSnapshot();
+        });
+
+        test('Annotating bound Action/Function paramter', () => {
+            // arrange
+            const target: Target = {
+                type: 'target',
+                name: 'AdminService.addRating(AdminService.Books)/stars',
+                range: {
```

Comment:
Ranges are optional and not used. We should remove them to make the test a bit more compact.